### ISSUE TITLE
[rfe_id:138][crit:high][vendor:cnv-qe@redhat.com][level:component] Upload tests [It] [test_id:3993] Upload image to data volume and verify retry count 

### ISSUE THREAD

**Issue by awels** [kubevirt/containerized-data-importer#1382]
**Is this a BUG REPORT or FEATURE REQUEST?**:

> Uncomment only one, leave it on its own line:
>
/kind bug
> /kind enhancement

**What happened**:
Test failure that should have passed.
**What you expected to happen**:

**How to reproduce it (as minimally and precisely as possible)**:

**Anything else we need to know?**:

**Environment**:
- CDI version (use `kubectl get deployments cdi-deployment -o yaml`): master
- Kubernetes version (use `kubectl version`): 1.17
- Cloud provider or hardware configuration: kubevirtci lane
- Install tools:
- Others:

**Comment by awels** [comment]
Found in https://prow.apps.ovirt.org/view/gcs/kubevirt-prow/pr-logs/pull/kubevirt_containerized-data-importer/1342/pull-containerized-data-importer-e2e-k8s-1.17-ceph/1306706383273463808

This seems to be the issue in the upload server pod
```
I0917 23:43:23.925161       1 uploadserver.go:64] Upload destination: /data/disk.img
I0917 23:43:23.925274       1 uploadserver.go:66] Running server on 0.0.0.0:8443
I0917 23:43:27.144150       1 uploadserver.go:405] Content type header is ""
I0917 23:43:27.144186       1 data-processor.go:302] Calculating available size
I0917 23:43:27.144222       1 data-processor.go:314] Checking out file system volume size.
I0917 23:43:27.144252       1 data-processor.go:322] Request image size not empty.
I0917 23:43:27.144279       1 data-processor.go:327] Target size 96239616.
I0917 23:43:27.144383       1 format-readers.go:110] constructReaders: checking compression and archive formats
I0917 23:43:27.144439       1 data-processor.go:224] New phase: TransferDataFile
I0917 23:43:27.151772       1 util.go:161] Writing data...
I0917 23:43:27.283623       1 data-processor.go:224] New phase: Resize
I0917 23:43:27.285335       1 data-processor.go:256] Available space in dataFile: -1
I0917 23:43:27.285347       1 data-processor.go:258] Resizing image
I0917 23:43:27.285377       1 prlimit.go:128] Setting CPU limit to 30
I0917 23:43:27.285962       1 prlimit.go:159] Setting Address space limit to 1073741824
W0917 23:43:27.308221       1 data-processor.go:289] Available space less than requested size, resizing image to available space 96239616.
I0917 23:43:27.308236       1 data-processor.go:295] Expanding image size to: 96239616
E0917 23:43:27.308347       1 data-processor.go:221] open /dev/null: operation not permitted
Couldn't start qemu-img
kubevirt.io/containerized-data-importer/pkg/system.executeWithLimits
	pkg/system/prlimit.go:151
kubevirt.io/containerized-data-importer/pkg/system.ExecWithLimits
	pkg/system/prlimit.go:111
kubevirt.io/containerized-data-importer/pkg/image.(*qemuOperations).Resize
	pkg/image/qemu.go:147
kubevirt.io/containerized-data-importer/pkg/importer.ResizeImage
	pkg/importer/data-processor.go:296
kubevirt.io/containerized-data-importer/pkg/importer.(*DataProcessor).resize
	pkg/importer/data-processor.go:259
kubevirt.io/containerized-data-importer/pkg/importer.(*DataProcessor).ProcessDataWithPause
	pkg/importer/data-processor.go:213
kubevirt.io/containerized-data-importer/pkg/importer.(*DataProcessor).ProcessData
	pkg/importer/data-processor.go:153
kubevirt.io/containerized-data-importer/pkg/uploadserver.newUploadStreamProcessor
	pkg/uploadserver/uploadserver.go:450
kubevirt.io/containerized-data-importer/pkg/uploadserver.(*uploadServerApp).uploadHandler.func1
	pkg/uploadserver/uploadserver.go:412
net/http.HandlerFunc.ServeHTTP
	GOROOT/src/net/http/server.go:2007
net/http.(*ServeMux).ServeHTTP
	GOROOT/src/net/http/server.go:2387
kubevirt.io/containerized-data-importer/pkg/uploadserver.(*uploadServerApp).ServeHTTP
	pkg/uploadserver/uploadserver.go:287
net/http.serverHandler.ServeHTTP
	GOROOT/src/net/http/server.go:2802
net/http.(*conn).serve
	GOROOT/src/net/http/server.go:1890
runtime.goexit
	src/runtime/asm_amd64.s:1357
Error resizing image /data/disk.img
kubevirt.io/containerized-data-importer/pkg/image.(*qemuOperations).Resize
	pkg/image/qemu.go:149
kubevirt.io/containerized-data-importer/pkg/importer.ResizeImage
	pkg/importer/data-processor.go:296
kubevirt.io/containerized-data-importer/pkg/importer.(*DataProcessor).resize
	pkg/importer/data-processor.go:259
kubevirt.io/containerized-data-importer/pkg/importer.(*DataProcessor).ProcessDataWithPause
	pkg/importer/data-processor.go:213
kubevirt.io/containerized-data-importer/pkg/importer.(*DataProcessor).ProcessData
	pkg/importer/data-processor.go:153
kubevirt.io/containerized-data-importer/pkg/uploadserver.newUploadStreamProcessor
	pkg/uploadserver/uploadserver.go:450
kubevirt.io/containerized-data-importer/pkg/uploadserver.(*uploadServerApp).uploadHandler.func1
	pkg/uploadserver/uploadserver.go:412
net/http.HandlerFunc.ServeHTTP
	GOROOT/src/net/http/server.go:2007
net/http.(*ServeMux).ServeHTTP
	GOROOT/src/net/http/server.go:2387
kubevirt.io/containerized-data-importer/pkg/uploadserver.(*uploadServerApp).ServeHTTP
	pkg/uploadserver/uploadserver.go:287
net/http.serverHandler.ServeHTTP
	GOROOT/src/net/http/server.go:2802
net/http.(*conn).serve
	GOROOT/src/net/http/server.go:1890
runtime.goexit
	src/runtime/asm_amd64.s:1357
Resize of image failed
kubevirt.io/containerized-data-importer/pkg/importer.(*DataProcessor).resize
	pkg/importer/data-processor.go:261
kubevirt.io/containerized-data-importer/pkg/importer.(*DataProcessor).ProcessDataWithPause
	pkg/importer/data-processor.go:213
kubevirt.io/containerized-data-importer/pkg/importer.(*DataProcessor).ProcessData
	pkg/importer/data-processor.go:153
kubevirt.io/containerized-data-importer/pkg/uploadserver.newUploadStreamProcessor
	pkg/uploadserver/uploadserver.go:450
kubevirt.io/containerized-data-importer/pkg/uploadserver.(*uploadServerApp).uploadHandler.func1
	pkg/uploadserver/uploadserver.go:412
net/http.HandlerFunc.ServeHTTP
	GOROOT/src/net/http/server.go:2007
net/http.(*ServeMux).ServeHTTP
	GOROOT/src/net/http/server.go:2387
kubevirt.io/containerized-data-importer/pkg/uploadserver.(*uploadServerApp).ServeHTTP
	pkg/uploadserver/uploadserver.go:287
net/http.serverHandler.ServeHTTP
	GOROOT/src/net/http/server.go:2802
net/http.(*conn).serve
	GOROOT/src/net/http/server.go:1890
runtime.goexit
	src/runtime/asm_amd64.s:1357
Unable to resize disk image to requested size
kubevirt.io/containerized-data-importer/pkg/importer.(*DataProcessor).ProcessDataWithPause
	pkg/importer/data-processor.go:215
kubevirt.io/containerized-data-importer/pkg/importer.(*DataProcessor).ProcessData
	pkg/importer/data-processor.go:153
kubevirt.io/containerized-data-importer/pkg/uploadserver.newUploadStreamProcessor
	pkg/uploadserver/uploadserver.go:450
kubevirt.io/containerized-data-importer/pkg/uploadserver.(*uploadServerApp).uploadHandler.func1
	pkg/uploadserver/uploadserver.go:412
net/http.HandlerFunc.ServeHTTP
	GOROOT/src/net/http/server.go:2007
net/http.(*ServeMux).ServeHTTP
	GOROOT/src/net/http/server.go:2387
kubevirt.io/containerized-data-importer/pkg/uploadserver.(*uploadServerApp).ServeHTTP
	pkg/uploadserver/uploadserver.go:287
net/http.serverHandler.ServeHTTP
	GOROOT/src/net/http/server.go:2802
net/http.(*conn).serve
	GOROOT/src/net/http/server.go:1890
runtime.goexit
	src/runtime/asm_amd64.s:1357
E0917 23:43:27.308521       1 uploadserver.go:418] Saving stream failed: Unable to resize disk image to requested size: Resize of image failed: Error resizing image /data/disk.img: Couldn't start qemu-img: open /dev/null: operation not permitted
```

**Comment by omeryahud** [comment]
Happened again here: https://prow.apps.ovirt.org/view/gcs/kubevirt-prow/pr-logs/pull/kubevirt_containerized-data-importer/1342/pull-containerized-data-importer-e2e-k8s-1.17-ceph/1306706383273463808
